### ISSUE TITLE
Make instructions of what to do clearer when opening the app for the first time

### ISSUE THREAD

**Issue by krisnaer** [trustlines-network/feedback#216]
### Basic information:

* OS (Android / iOS / both):  both
* Version: N/A

### Issue description:

When I open the app for the first time, I don't know what to do. I can see a button that says add contacts, yet I am not sure why I should add contacts. 

### Screenshots or mock-ups
![3054b0da-9137-4f0c-b8ef-f3a498a2834d](https://user-images.githubusercontent.com/29100596/45223749-64d1d580-b2b8-11e8-833b-a463a82951de.jpeg)

### Possible solution

The text could instead say:

"Join a Trustline Network" or "create your first trustline" or a similar statement which indicates that the user should create a trustline since this is what we would like them to do.

**Comment by seichris** [comment]
Agree. I would simply grey out everything but the _add contact_ button. And write "Join a Trustlines Network by adding your first contact"

This issue is linked to enabling the user to choose networks.
